trieste/models/gpflow/architectures.py

```diff
@@ -0,0 +1,228 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" This file contains builders for some GPflow models supported in Trieste. """
+
+from __future__ import annotations
+
+import math
+from typing import Optional
+
+import gpflow
+import tensorflow as tf
+import tensorflow_probability as tfp
+from gpflow.models import GPR, SGPR, SVGP, VGP
+
+from ...data import Dataset
+from ...space import Box
+from ...types import TensorType
+
+
+def build_gpr(
+    data: Dataset,
+    search_space: Box,
+    kernel_priors: bool = True,
+    noise_free: bool = False,
```

Comment:
This should be `trainable_noise`, not `noise_free`. To get the noise free setting you set `lik_var` to 1e-5 and  trainable to False.